index.js

```diff
@@ -325,6 +332,76 @@ class GithubScm extends Scm {
             url: branchUrl
         }));
     }
+
+    /**
+     * Given a SCM webhook payload & its associated headers, aggregate the
+     * necessary data to execute a Screwdriver job with.
+     * @method parseHook
+     * @param  {Object}  webhookPayload  The webhook payload received from the
+     *                                   SCM service.
+     * @param  {Object}  payloadHeaders  The request headers associated with the
+     *                                   webhook payload
+     * @return {Object}                  A key-map of data related to the received
+     *                                   payload
+     */
+    parseHook(webhookPayload, payloadHeaders) {
+        const type = payloadHeaders['x-github-event'];
+        const checkoutUrl = hoek.reach(webhookPayload, 'repository.ssh_url');
+
+        switch (type) {
+        case 'pull_request': {
+            let action = hoek.reach(webhookPayload, 'action');
+            const prNum = hoek.reach(webhookPayload, 'pull_request.number');
+
+            if (!SUPPORTED_PR_ACTIONS.includes(action)) {
+                action = 'closed';
```

Comment:
That's the default now: https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/webhooks/github.js#L238